bigchaindb/web/server.py

```diff
@@ -88,6 +91,7 @@ def create_server(settings):
     if not settings.get('threads'):
```

Comment:
Mmm true, `settings` could be passed in as arguments by destructing them from the bdb config, but then we'd have to reconstruct them into a dict when we create the gunicorn app (see https://github.com/bigchaindb/bigchaindb/pull/750/files#diff-9943b8877e792a45dc46b4e483e1c02eR35).